### ISSUE TITLE
Performance testing of Mogadishu

### ISSUE THREAD

**Issue by blackwatchint** [blackwatchint/blackwatchint#65]
There have been a number of reports of poor performance on the Mogadishu map. A test needs to be done to determine how much of a performance difference there is between Mogadishu and a benchmark map (i.e. Altis).

**Comment by Kentersnoodlefactory** [comment]
From only running around in Altis i got 30-35 fps, with full view distance very high terrain and pretty high graphic settings, about same on Mogadishu. I also per request tried to do a 2 squad vs 2 squads AI battle, which shot my fps down to 12-17 fps on Mogadishu, slightly better on Altis varying from 16-20 fps.

**Comment by Fourjays** [comment]
I did a test using 4 AI squads vs 4 AI squads at natural chokepoints within the city of Mogadishu. I repeated the exercise on Altis, using the streets of Kavala.

Mogadishu ran between 22FPS and 25FPS during the battle, mostly at 22FPS. Altis also ran between 22FPS and 25FPS, but was closer to 25FPS most of the time. Without any battles going on, Mogadishu was running at around 50-55FPS and Altis at 60FPS.

**Comment by karlkarl93** [comment]
I used 2 AI squads and did the test this way: I first used altis and went to 3 cities, Pyrgos (Medium), Gravia (Small) and Kavala (Large) and I did tests standing still in different places and running around. I did that with 2 AI squads fighting and with nothing else but me.

I averaged 55 FPS on Altis in all of the towns when running or standing with nothing spawned and with 2 AI squads fighting I got 36-24 FPS, average was 28 FPS.

Then I did the same with different parts of Mogadishu. I got around 50 FPS in Mogadishu with nothing spawned and with 2 AI squads fighting I got 30-20 FPS average was 24 FPS. 

To me it seems like mogadishu has aprox. -5 FPS from any Altis average. From experience in ops, I had 19 FPS with heavy fighting and smoke everywhere in the middle of the town and around 22-24 most of the op.
